src/pixel/fiddler.ts

```diff
@@ -40,10 +39,10 @@ function _itemsLimiter(state: State): State {
   return {}
 }
 
-const fiddlers = [_provided, _itemsLimiter]
+const fiddlers = [provided, itemsLimiter]
 
-export function fiddle(state: State): State {
-  const reducer = (accumulator: State, func: (current: State) => State) => {
+export function fiddle<A extends { eventSource?: Record<string, unknown> }>(state: A): A & { hashedEmail?: string[] } {
```

Comment:
```suggestion
export function fiddle<A extends { eventSource?: Record<string, unknown> }>(state: A): A & { hashedEmail?: HashedEmail } {
```

Reply:
it's not a list of HashedEmails here but rather a list of strings. the typing is correct.